apis/hive/v1/clusterdeployment_types.go

```diff
@@ -400,6 +400,10 @@ const (
 	// ProvisionStoppedCondition is set when cluster provisioning is stopped
 	ProvisionStoppedCondition ClusterDeploymentConditionType = "ProvisionStopped"
 
+	// Provisioned is True when a cluster is installed; False while it is provisioning or deprovisioning.
+	// The Reason indicates where it is in that lifecycle.
+	ProvisionedCondition ClusterDeploymentConditionType = "Provisioned"
```

Comment:
Also, can we eliminate `ProvisionStopped` condition since we have the corresponding reason in the new condition? It would be one less condition we have on clusterdeployment

Reply:
> add it to the corresponding array

Nice catch, done.

> one less condition

I definitely like the idea of consolidating conditions...

...but I think ProvisionStopped still has value in terms of being able to `oc wait` for it to go `True` as an indicator that the CD is unrecoverable.

Happy to discuss this further -- but I think we could consider it separately as a followon?